src/platform/landing-pages/dev-template.ejs

```diff
@@ -0,0 +1,168 @@
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="HandheldFriendly" content="True">
+    <meta name="MobileOptimized" content="320">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <script src="/js/record-event.js"></script>
+    <script src="/js/settings.js"></script>
+
+
+    <!-- Icons -->
+    <link href="/img/design/icons/apple-touch-icon.png" rel="apple-touch-icon-precomposed" />
+    <link href="/img/design/icons/apple-touch-icon-72x72.png" rel="apple-touch-icon-precomposed" sizes="72x72" />
+    <link href="/img/design/icons/apple-touch-icon-114x114.png" rel="apple-touch-icon-precomposed" sizes="114x114" />
+    <link href="/img/design/icons/apple-touch-icon-152x152.png" rel="apple-touch-icon-precomposed" sizes="144x144" />
+    <link rel="shortcut icon" href="/img/design/icons/favicon.ico" />
+
+    <meta name="msapplication-TileImage" content="/img/design/icons/apple-touch-icon-114x114">
+    <meta name="msapplication-TileColor" content="#144073">
+
+    <!-- Preload main fonts -->
+
+    <link rel="preload" href="/generated/sourcesanspro-bold-webfont.woff2" as="font" type="font/woff2" crossorigin>
+    <link rel="preload" href="/generated/sourcesanspro-regular-webfont.woff2" as="font" type="font/woff2" crossorigin>
+    <link rel="preload" href="/generated/bitter-bold.woff2" as="font" type="font/woff2" crossorigin>
+    <link rel="preload" href="/generated/fa-solid-900.woff2" as="font" type="font/woff2" crossorigin>
+
+    <!-- CSS -->
+    <link rel="stylesheet" href="/generated/style.css">
+    <link rel="stylesheet" href="/generated/<%= entryName %>.css">
+
+
+    <script defer nomodule src="/generated/polyfills.entry.js"></script>
+    <script defer src="/generated/vendor.entry.js"></script>
+    <script defer src="/generated/<%= entryName %>.entry.js"></script>
+
+    <script type="text/javascript">
+      window.VetsGov = window.VetsGov || {};
+      window.VetsGov.headerFooter = <%= JSON.stringify(headerFooterData) %>;
```

Comment:
Note that the templating engine is different. This isn't Liquid anymore. It's _basically_ EJS, but _actually_ lodash templates. See the `html-webpack-plugin`'s ["template option" doc](https://github.com/jantimon/html-webpack-plugin/blob/master/docs/template-option.md) for more details.